modeshape-parent/pom.xml

```diff
@@ -188,7 +188,9 @@
         <version.postgresql.9>9.2-1002.jdbc4</version.postgresql.9>
         <version.postgresql.8>8.4-702.jdbc4</version.postgresql.8>
         <version.mysql>5.1.38</version.mysql>
+        <version.mysql.server>5.7</version.mysql.server>
```

Comment:
again, this should be merged with the above property so we only use one version of the driver. Driver versions should be backwards compatible